### PR TITLE
Update jenkinsfile to build purecap FPGA kernels

### PR DESCRIPTION
**Pull request by qwattash**

### CODE REVIEW

**Comment by arichardson** [comment]
I was assuming you were going to build the fpga kernels with the minimal disk image, i.e. `cheribsd-mfs-root-kernel-${suffix}`.

**Comment by arichardson** [comment]
However, setting the mfs-image option might actually include it for the kernels with MFS_ROOT?

File: Jenkinsfile [hunk 1]
```diff
@@ -60,6 +60,22 @@ find cheribsd-test-results
     }
 }
 
+def buildFPGAKernels(params, String name, String suffix) {
+    stage("Building minimal disk image") {
+        sh "./cheribuild/jenkins-cheri-build.py --build disk-image-minimal-${suffix} ${params.extraArgs}"
+    }
+    stage("Building FPGA kernels") {
+        cheribuildProject(target: "cheribsd-${suffix}", architecture: suffix,
+            extraArgs: "--cheribsd/build-options=-s --cheribsd/no-debug-info --keep-install-dir --install-prefix=/rootfs --cheribsd-${suffix}/pure-cap-kernel --cheribsd-${suffix}/build-fpga-kernels --cheribsd-${suffix}/mfs-root-image $WORKSPACE/cherisdk/disk-image-minimal-${suffix} --skip-buildworld",
```
Comment (arichardson) [inline Jenkinsfile:69]:
```suggestion
            extraArgs: params.extraArgs + "--cheribsd-${suffix}/build-fpga-kernels --cheribsd-${suffix}/mfs-root-image $WORKSPACE/cherisdk/disk-image-minimal-${suffix} --skip-buildworld",
```

**Comment by qwattash** [comment]
@arichardson I can use the `cheribsd-mfs-root-kernel-${suffix}` target, for the purecap kernels it should not make much difference, however the `--build-fpga-kernels` option in the cheribsd-purecap project will build also other non-mfsroot kernels, which we should arguably test as well if we want to keep them?

**Pull request closed without merging**